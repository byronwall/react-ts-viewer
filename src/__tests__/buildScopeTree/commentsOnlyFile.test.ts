import { describe, it } from "vitest";
import { runScopeTreeSnapshotTest } from "./testUtils";

describe("buildScopeTree - Comments Only File", () => {
  it("should handle a file with only comments", () => {
    runScopeTreeSnapshotTest({
      snapshotIdentifier: "commentsOnlyFile",
      filePath: "commentsOnly.tsx", // Mock file path
      isFixture: false,
      inlineContent: "// This is a comment\n/* Another comment */",
    });
  });
});
