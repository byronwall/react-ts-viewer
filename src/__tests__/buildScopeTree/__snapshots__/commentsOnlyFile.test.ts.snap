// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`buildScopeTree - Comments Only File > should handle a file with only comments > commentsOnlyFile 1`] = `
{
  "category": "Program",
  "children": [],
  "id": "commentsOnly.tsx",
  "kind": 307,
  "label": "commentsOnly.tsx",
  "loc": {
    "end": {
      "column": 21,
      "line": 2,
    },
    "start": {
      "column": 0,
      "line": 1,
    },
  },
  "source": "// This is a comment
/* Another comment */",
  "value": 42,
}
`;
